cluster/gce/gci/configure-helper.sh

```diff
@@ -1588,6 +1592,10 @@ function start-kube-apiserver {
       if [[ -n "${ADVANCED_AUDIT_WEBHOOK_INITIAL_BACKOFF:-}" ]]; then
         params+=" --audit-webhook-initial-backoff=${ADVANCED_AUDIT_WEBHOOK_INITIAL_BACKOFF}"
       fi
+      # Truncating backend parameters
```

Comment:
This gets unwieldy. Please extract common flags into single if statements. It seems to me that the difference between to branches is only with second component of the flag name.

Reply:
All those flags, are different. In first branch we control plugins for log backend in the second for webhook. Each of them is controlled by a separate env variable. 

Actually I see one bug in all this code (not in this PR), that we don't allow both backends at the same time (inside of the apiserver there is no limitations). But this problem I would prefer to fix in different PR.